public/app/common/components/device-status/device-status-text/style.scss

```diff
@@ -1,6 +1,7 @@
 @import "components-theme";
 
-.cc-device-state-text {
```

Comment:
Wouldn't the correct wording be _state_? See also http://english.stackexchange.com/questions/12958/status-vs-state